src/main/java/Deadline.java

```diff
@@ -0,0 +1,10 @@
+public class Deadline extends Task{
+    protected String by;
+    public Deadline(String description, String by){
+        super(description);
+        this.by = by;
+    }
+    public String toString(){
+        return "[D]" + super.toString() + "by: " + by + ")";
```

Comment:
It might be more readable if you add the @Override before overriding toString() method.